doc/magic_cards_notes.md

```diff
@@ -1137,6 +1204,8 @@ Ultralight mode, 10b UID
 
 UID and ATQB are configured according to block0 with a (14a) backdoor write.
 
+👉 hf_mf_ultimatecard.lua does not support 14443B commands yet.
```

Comment:
the script is only one way,  not that emphased.   
I tend to only want to see how to do it,  not getting information that it can'tbe done.

Reply:
Removed